src/components/BPJsonGenerator/NodesForm.js

```diff
@@ -176,6 +176,8 @@ const NodesForm = ({ nodes, nodeIndex, onSubmit, openModal, setOpenModal }) => {
     setCurrentNode(nodes[nodeIndex] || defaultNode)
   }, [nodes, nodeIndex])
 
+  console.log(nodes)
```

Comment:
@JustinCast we should avoid commit `console.log` used for debug

Reply:
Thanks